Projects/hCore/Sources/FeatureFlags.swift

```diff
@@ -0,0 +1,111 @@
+import Foundation
+import UnleashProxyClientSwift
+import hGraphQL
+
+public class FeatureFlags {
```

Comment:
On Android we have a 
```
internal interface FeatureFlagProvider {
  suspend fun isFeatureEnabled(feature: Feature): Boolean
}
```

which is targeting our local feature flags for develop, and unleash for staging and prod. 

See 
https://github.com/HedvigInsurance/android/blob/1792e9f6381d48ea066c95a10948101c4e384a45/app/hanalytics/hanalytics-feature-flags-public/src/main/kotlin/com/hedvig/android/hanalytics/featureflags/flags/HAnalyticsFeatureFlagProvider.kt#L4

https://github.com/HedvigInsurance/android/blob/1792e9f6381d48ea066c95a10948101c4e384a45/app/hanalytics/hanalytics-feature-flags-public/src/main/kotlin/com/hedvig/android/hanalytics/featureflags/flags/DevFeatureFlagProvider.kt

This makes it really easy to create local feature flags, and merge code directly to develop instead of having long-running feature branches. Something you can consider as well!

Reply:
Nice! Did some changes related to this